src/lib/net.py

```diff
@@ -23,19 +23,27 @@
 from .utils import ROOT
 
 
-def download_snapshot(url: Union[Path, str], ext: str = None, offline: bool = False) -> str:
+def download_snapshot(url: str, output_folder: Path, ext: str = None, offline: bool = False) -> str:
     """
     This function downloads a file into the snapshots folder and outputs the
```

Comment:
"returns" might be better phrasing than "outputs" since the latter might be confused with stdout or writing to a file.

Reply:
I'm not sure I agree. Every other similar project has more or less agreed on the "outputs" name as a convention.

Reply:
For describing what the function does, I prefer the term "returns" as it is more specific.   Oscar, when you say projects have agreed on the "outputs" name, do you mean for use in describing the return value of a function, or are you referring to something else like the directory name?